modules/javafx.controls/src/test/java/test/javafx/scene/control/skin/TreeTableColumnHeaderTest.java

```diff
@@ -0,0 +1,164 @@
+/*
+ * Copyright (c) 2018, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
Replace "2018, 2021" with just "2022" as this is newly introduced test file.

Reply:
done